mvp-spec/prebid-implementation.md

```diff
@@ -0,0 +1,691 @@
+<!-- ⚠️ GENERATED CONTENT - DO NOT MODIFY DIRECTLY ⚠️ -->
+```mermaid
+sequenceDiagram
+    participant User
+    participant Browser
+    participant Publisher
+    participant SSP
+    participant DSP
+    participant AdServer
+
+    User->>Browser: Ask for a webpage
+    Browser->>Browser: Generate or Fetch<br /> PAF Data & Pref
+    Browser->>Publisher: Request Transmission Seed
+    Publisher->>Browser: Return Seed
+    Browser->>Browser: Prebid.js enriches<br /> PAF Data for SSP
+    Browser->>SSP: Generate and send an <br /> AdRequest + Transmission
+    SSP->>SSP: Sign Transmission
+    SSP->>DSP: Send AdRequest + Transmission
+    DSP->>SSP: Provide Addressable Content <br /> and a Transmission Response
+    SSP->>Browser: Provide the Addressable Contents<br />and Transmission Responses
+    Browser->>AdServer: AdRequest with Prebid.js key values
+    AdServer->>Browser: Return Prebid.js callback
+    Browser->>User: Render Prebid.js ad and provide audit log
+```
+<!--partial-end-->
+
+## Implementation Details
+
+PAF doesn't standardize the way of offering inventory because 
+there are already mechanisms sush as OpenRTB for these cases.
+However to offer PAF the Publisher must implement additional features.
+
+To understand the steps, it is important to overview how to generate the data and
+the relationships between them:
+* A Publisher offers *multiple* placements - one for each for 
+Addressable Contents - via Prebid.js.
+* A Publisher Server generates a Seed for each PAF Data transaction
+* Prebid.js sends a Request for opportunites and *one* Transaction Request
+* A Supplier generates and sends *one* Transaction Response per Impression Opportunity
+
+
+### Prebid.js userId module
+
+Prebid.js already offers identity submodules and a PAF sub module will be added.
+This module will be very straight-forward and will simply fetch the PAF data from the
+window using the [PAF frontend](https://github.com/criteo/paf-mvp-implementation/tree/main/paf-mvp-frontend). The module will call
+`PAF.getIdsAndPreferences()` to get the PAF Data. It will next call `PAF.generateSeeds(1)`
+to retrieve a transmission seed. These data will be combined and offered to all
+configured bidders in the transmission request format:
+
+<!--partial-begin { "files": [ "transmission-requests.json" ], "block": "json" } -->
+<!-- ⚠️ GENERATED CONTENT - DO NOT MODIFY DIRECTLY ⚠️ -->
+```json
+{
+    "data": {
+        "identifiers": [
+            {
+                "version": 0,
+                "type": "prebid_id",
+                "value": "7435313e-caee-4889-8ad7-0acd0114ae3c",
+                "source": {
+                    "domain": "operator0.com",
+                    "timestamp": 1639580000,
+                    "signature": "12345_signature"
+                }
+            }
+        ],
+        "preferences": {
+            "version": 0,
+            "data": { 
+                "opt_in": true 
```

Comment:
@RomainLofaso this example is still using the old naming, same for prebid_id. And it has been reused by @bwschmidt in his own examples below.
I think this should all be updated ASAP to avoid confusion, see https://github.com/criteo/prebid-addressability-framework-proposal/issues/59